httpclient5/src/main/java/org/apache/hc/client5/http/impl/nio/PoolingAsyncClientConnectionManager.java

```diff
@@ -315,10 +315,6 @@ public void completed(final PoolEntry<HttpRoute, ManagedAsyncClientConnection> p
                                             })), Command.Priority.IMMEDIATE);
                                             return;
```

Comment:
@rschmitt I guess `return` here can also be removed. Looks good otherwise.

Reply:
Won't that cause `leaseCompleted(poolEntry)` to be called immediately (line 321) after the PingCommand is submitted, and then a second time after the PingCommand finishes?

Reply:
@rschmitt My bad. Disregard.